src/OrbitGl/AccessibleTrack.cpp

```diff
@@ -92,43 +55,37 @@ AccessibleTrack::AccessibleTrack(Track* track, const TimeGraphLayout* layout)
     : AccessibleCaptureViewElement(track, track->GetName(),
                                    orbit_accessibility::AccessibilityRole::Grouping),
       track_(track),
-      fake_tab_(std::make_unique<FakeTrackTab>(track, layout)),
-      fake_timers_pane_(std::make_unique<FakeTimerPane>(track, layout, fake_tab_.get())) {}
+      fake_timers_pane_(std::make_unique<FakeTimerPane>(track, layout)) {}
 
 int AccessibleTrack::AccessibleChildCount() const {
   ORBIT_CHECK(track_ != nullptr);
 
   // Only expose the "Timer" pane if any timers were rendered in the visible field
   if (track_->GetVisiblePrimitiveCount() > 0) {
-    return static_cast<int>(track_->GetNonHiddenChildren().size()) + 2;
+    return static_cast<int>(track_->GetNonHiddenChildren().size()) + 1;
```

Comment:
Now, what is the +1 for?

Reply:
Yep. I've gotten rid of one fake element, but not all of them yet. I can add a comment.

Reply:
Thanks. Adding a comment would be nice.

Reply:
Extended the comment above